backup_and_restore/application_backup_and_restore/installing/uninstalling-oadp.adoc

```diff
@@ -6,4 +6,16 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You uninstall the OpenShift API for Data Protection (OADP) by deleting the OADP Operator. See xref:../../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-cluster[Deleting Operators from a cluster] for details.
+You uninstall the OpenShift API for Data Protection (OADP) by deleting the OADP Operator.
+
+[id="uninstalling-oadp-web-console"]
+== Deleting Operators from a cluster
+
+Cluster administrators can delete installed Operators from a selected namespace by using the web console. See xref:../../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster-using-web-console[Deleting Operators from a cluster using the web console].
```

Comment:
Is this link right? It seems to go to the top of the file rather than to the module. Should it go to: #olm-deleting-operators-from-a-cluster-using-web-console_olm-deleting-operators-from-a-cluster

Reply:
@mburke5678 - these are the best links for removing an operator from OpenShift... I know they should not be, but it is what it is, so it is.